src/patients/edit/EditPatient.tsx

```diff
@@ -104,6 +113,77 @@ const EditPatient = () => {
         }))
       }
     }
+    if (patient.suffix) {
+      if (regexContainsNumber.test(patient.suffix)) {
```

Comment:
I am not sure that we have to block numbers from being added to suffix and prefix. I think that some cultures can use numbers in this fields. @jackcmeyer what do you think?

Reply:
Not sure about prefix but now that you mention it, I think it is possible in a suffix. However usually in Roman numerals...

Reply:
OK. We can try to implement this and if, in the future, someone needs numeric values we can fix it accordingly. :)